### PR TITLE
Add removal and addition of books feature

### PR DESCRIPTION
**Pull request by kene-creator**
### Feature added

- Addition of books
- Removal of added books
- Storing the books to the browser's local storage

**PS: My coding partner is unavailable**

### CODE REVIEW

**Comment by kene-creator** [comment]
The dashboard is showing changes required but the last reviewer seems to have not reviewed the code or made any suggested changes

File: script.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+const removeBtn = document.createElement('button');
+const addbtn = document.querySelector('.book__add-btn');
+const bookTitle = document.querySelector('.book__title');
+const bookOwner = document.querySelector('.book__owner');
+const book = document.querySelector('.book');
+const bookAuthor = document.getElementsByClassName('book__author');
+const bookName = document.getElementsByClassName('book__name');
+const bookItem = document.querySelector('.book__item');
+
```
Comment (thecodechaser) [inline script.js:9]: - [ ] Please make sure that added books are stored in local storage as requested in the project requirements. After refreshing the page users should be able to see the previously added books.
image

File: script.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+const bookList = [];
+const bookDisplay = [];
+
+addbtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  const bookInfo = {
+    title: bookTitle.value,
+    author: bookOwner.value,
+  };
+
```
Comment (thecodechaser) [inline script.js]: - [ ] [optional] I would suggest clearing the form fields after adding a book to the collection, It will improve the user experience.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @kene-creator and Team👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
Hi @kene-creator, 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**